crates/re_viewer/src/ui/selection_panel.rs

```diff
@@ -559,6 +561,12 @@ fn container_top_level_properties(
                             }
                         });
 
+                    // TODO(jleibs): Manually marking edited like this is way too error prone.
+                    // Need to detect this in a better way.
+                    if original_layout != grid.layout {
+                        viewport.edited = true;
+                    }
```

Comment:
style: this can be made shorter

```suggestion
                    viewport.edited |= original_layout != grid.layout;
```